Algorithm/Bit_manipulation/readme.md

```diff
@@ -52,4 +52,8 @@ There are different bitwise operations used in the bit manipulation. Bit operati
 
 
 ## Questions :
-- Brian Kernighan's Algorithm ----> [C++](/Code/C++/Brian_Kernighan.cpp) 
\ No newline at end of file
+
```

Comment:
Plz bring it back to the original approved state.

Reply:
Do this change, we'll merge your PR then.

Reply:
Done